### PR TITLE
Tests should work

### PR DESCRIPTION
**Pull request by hirdbluebird**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request.
If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to static food data?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] refactor
- [x] documentation
- [ ] other

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

Are tests included?
- [ ] yes (*bugfixes and features will not be merged without tests*)
- [ ] no

Breaking Changes?
- [ ] yes (*breaking changes will not be merged unless absolutely necessary*)
- [x] no

List any relevant issue numbers:

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: src/projects/MealCalendar/mealCalendar.js [hunk 1]
```diff
@@ -34,13 +34,14 @@ const getDishByMonth = function (monthNumb) {
   if (monthNumb < 1 || monthNumb > 5) return []
   const totalWeeks = monthNumb * 4 + 1
   const weeks = _.range(totalWeeks - 4, totalWeeks)
-  return _.map(weeks, week => this.getDishByWeek(`week${week}`)[0])
+  return _.map(weeks, week => this.getDishByWeek(`week ${week}`)[0])
```
Comment (atherdon) [inline src/projects/MealCalendar/mealCalendar.js]: i don't like this long line - can you split it into few variables/steps?

File: src/projects/GroceriStar/groceristar.js [hunk 2]
```diff
@@ -33,9 +33,9 @@ const getUsers = function () {
   // return __get(users)
 }
 
-// const getDepartments = function () {
-//   return __get(departments)
-// }
+const getDepartments = () => {
+  return __get(departments)
```
Comment (atherdon) [inline src/projects/GroceriStar/groceristar.js]: if we import departments at the beginning - we actually don't need these `__get` and `getXXX()` methods now

**Comment by atherdon** [comment]
You see - I'm helping you!

**Comment by atherdon** [comment]
my comment was not published and i cannot see that line of code.
we have tests and you create a function `isArray` - i want to have another name for that function, because it's confusing and look similar to `Array.isArray` method

File: src/projects/UnitConvertion/test/files.test.js [hunk 3]
```diff
@@ -60,27 +60,26 @@ describe('testing objects are not empty', () => {
 describe('testing object returns array', () => {
   const isArray = (object) => {
```
Comment (atherdon) [inline src/projects/UnitConvertion/test/files.test.js]: This is a function that i talk about

Comment (atherdon) [reply src/projects/UnitConvertion/test/files.test.js]: i think it resolved

**Comment by atherdon** [comment]
tell me when i need to merge it. i can do it right now

File: src/projects/GroceriStar/groceristar.js [hunk 4]
```diff
@@ -100,14 +100,17 @@ const getGroceriesWithDepIngKey = () => {
...
   return {
     'name': 'name',
     'groceryId': 'groceryId',
     'messages': {},
     'departments': {
-      'name': ''
+      'id': departmentId,
+      'name': '',
+      'type': '',
+      'ingridients': {}
```
Comment (ashleycribb) [inline src/projects/GroceriStar/groceristar.js]: String 'ingridients' should be 'ingredients'

Comment (atherdon) [reply src/projects/GroceriStar/groceristar.js]: Thanks!

**Comment by hirdbluebird** [comment]
@atherdon approve this pull request

**Comment by atherdon** [comment]
publish at npm as well

File: src/projects/GroceriStar/groceristar.js (hunk 4 above)
Comment (atherdon) [reply src/projects/GroceriStar/groceristar.js]: done

**Pull request merged**